unstructured/ingest/cli/interfaces.py

```diff
@@ -59,6 +61,45 @@ def add_cli_options(cmd: click.Command) -> None:
         pass
 
 
+class CliRetryStrategyConfig(RetryStrategyConfig, CliMixin):
```

Comment:
depending on sequencing of PRs, just want to make sure this is also captured in documentation at some point.